x-pack/filebeat/module/kubernetes/k8s-ingest.yaml

```diff
@@ -0,0 +1,142 @@
+      username: ${ELASTICSEARCH_USERNAME}
+      password: ${ELASTICSEARCH_PASSWORD}
+
+    setup.kibana:
+      host: '${KIBANA_HOST:kibana}:${KIBANA_PORT:5601}'
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: ingest
+  namespace: kube-system
+  labels:
+    k8s-app: ingest
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: ingest
+    spec:
+      serviceAccountName: ingest
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: ingest
+        image: docker.elastic.co/beats/filebeat:8.0.0
+        args: [
+          "sh", "-c", "filebeat setup -e --modules kubernetes -c /etc/filebeat-k8s.yml && filebeat -e -c /etc/filebeat-k8s.yml"
+        ]
+        env:
+        - name: ELASTICSEARCH_HOST
+          value: 192.168.99.1
+        - name: ELASTICSEARCH_USERNAME
+          value: elastic
+        - name: ELASTICSEARCH_PASSWORD
+          value: changeme
+        - name: KIBANA_HOST
+          value: 192.168.99.1
+        - name: ELASTIC_CLOUD_ID
+          value:
+        - name: ELASTIC_CLOUD_AUTH
+          value:
+        securityContext:
+          runAsUser: 0
+          # If using Red Hat OpenShift uncomment this:
+          #privileged: true
+        resources:
+          limits:
+            memory: 256Mi
+          requests:
+            cpu: 100m
+            memory: 128Mi
+        volumeMounts:
+        - name: config
+          mountPath: /etc/filebeat-k8s.yml
+          readOnly: true
+          subPath: filebeat-k8s.yml
+        - name: data
+          mountPath: /usr/share/filebeat/data
+        - name: varlibdockercontainers
+          mountPath: /var/lib/docker/containers
+          readOnly: true
+        #- name: varlogcontainers
+        #  mountPath: /var/log/containers
+        #  readOnly: true
+      volumes:
+      - name: config
+        configMap:
+          defaultMode: 0600
+          name: ingest-config
+      - name: varlibdockercontainers
+        hostPath:
+          path: /var/lib/docker/containers
+      # data folder stores a registry of read status for all files, so we don't send everything again on a Filebeat pod restart
+      - name: data
+        hostPath:
+          path: /var/lib/filebeat-data
+          type: DirectoryOrCreate
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: ingest
+subjects:
+- kind: ServiceAccount
+  name: ingest
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: ingest
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: ingest
+  labels:
+    k8s-app: ingest
+rules:
+- apiGroups: [""] # "" indicates the core API group
+  resources:
+  - namespaces
+  - pods
+  verbs:
+  - get
+  - watch
+  - list
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ingest
+  namespace: kube-system
+  labels:
+    k8s-app: ingest
+---
+
```

Comment:
There are already deploy files in `deploy/kubernetes`, could this be added there?